### ISSUE TITLE
HTTP Status 500 – Internal Server Error on fresh install

### ISSUE THREAD

**Issue by jw098** [open-osp/open-osp#168]
On a new install, after logging in for the first time and changing the default password, I get the following error: 
HTTP Status 500 – Internal Server Error. javax.servlet.ServletException: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]

This is on Ubuntu 22.04.

**Comment by warrendennis** [comment]
Some updates were missed in the repository.  It's fixed now.  Please try to reinstall from the beginning.  

Or if you have the skills to edit the database you can add this update: 

```ALTER TABLE ProviderPreferenceAppointmentScreenEForm ADD COLUMN eFormName varchar(255);```

If that doesn't work then please send us the entire stack trace for analysis.

**Comment by jw098** [comment]
I still get the same error after purging and reinstalling.

Message javax.servlet.ServletException: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]

Description The server encountered an unexpected condition that prevented it from fulfilling the request.

Exception

javax.servlet.ServletException: javax.servlet.ServletException: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]
	org.oscarehr.util.DbConnectionFilter.doFilter(DbConnectionFilter.java:69)
	org.oscarehr.util.UserActivityFilter.doFilter(UserActivityFilter.java:88)
	org.oscarehr.common.printing.PrivacyStatementAppendingFilter.doFilter(PrivacyStatementAppendingFilter.java:121)
	org.oscarehr.util.ResponseDefaultsFilter.doFilter(ResponseDefaultsFilter.java:102)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:199)

Root Cause

javax.servlet.ServletException: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]
	org.apache.struts.action.RequestProcessor.processException(RequestProcessor.java:523)
	org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:421)
	org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:224)
	org.apache.struts.action.ActionServlet.process(ActionServlet.java:1194)
	org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:432)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:239)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:215)
	oscar.oscarSecurity.LoginFilter.doFilter(LoginFilter.java:205)
	org.displaytag.filter.ResponseOverrideFilter.doFilter(ResponseOverrideFilter.java:125)
	org.oscarehr.util.LoggedInUserFilter.doFilter(LoggedInUserFilter.java:56)
	org.oscarehr.util.DbConnectionFilter.doFilter(DbConnectionFilter.java:66)
	org.oscarehr.util.UserActivityFilter.doFilter(UserActivityFilter.java:88)
	org.oscarehr.common.printing.PrivacyStatementAppendingFilter.doFilter(PrivacyStatementAppendingFilter.java:121)
	org.oscarehr.util.ResponseDefaultsFilter.doFilter(ResponseDefaultsFilter.java:102)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:199)

Root Cause

javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]
	org.hibernate.ejb.AbstractEntityManagerImpl.throwPersistenceException(AbstractEntityManagerImpl.java:614)
	org.hibernate.ejb.AbstractEntityManagerImpl.find(AbstractEntityManagerImpl.java:202)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:498)
	org.springframework.orm.jpa.SharedEntityManagerCreator$SharedEntityManagerInvocationHandler.invoke(SharedEntityManagerCreator.java:240)
	com.sun.proxy.$Proxy101.find(Unknown Source)
	org.oscarehr.common.dao.AbstractDao.find(AbstractDao.java:145)
	org.oscarehr.common.dao.AbstractDao$$FastClassByCGLIB$$d259da96.invoke(<generated>)
	net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191)
	org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)
	org.oscarehr.common.dao.ProviderPreferenceDao$$EnhancerByCGLIB$$c53e174c.find(<generated>)
	oscar.login.LoginAction.execute(LoginAction.java:334)
	org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:419)
	org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:224)
	org.apache.struts.action.ActionServlet.process(ActionServlet.java:1194)
	org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:432)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:239)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:215)
	oscar.oscarSecurity.LoginFilter.doFilter(LoginFilter.java:205)
	org.displaytag.filter.ResponseOverrideFilter.doFilter(ResponseOverrideFilter.java:125)
	org.oscarehr.util.LoggedInUserFilter.doFilter(LoggedInUserFilter.java:56)
	org.oscarehr.util.DbConnectionFilter.doFilter(DbConnectionFilter.java:66)
	org.oscarehr.util.UserActivityFilter.doFilter(UserActivityFilter.java:88)
	org.oscarehr.common.printing.PrivacyStatementAppendingFilter.doFilter(PrivacyStatementAppendingFilter.java:121)
	org.oscarehr.util.ResponseDefaultsFilter.doFilter(ResponseDefaultsFilter.java:102)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:199)

Root Cause

org.hibernate.exception.SQLGrammarException: could not initialize a collection: [org.oscarehr.common.model.ProviderPreference.appointmentScreenForms#999998]
	org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:90)
	org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	org.hibernate.loader.Loader.loadCollection(Loader.java:2022)
	org.hibernate.loader.collection.CollectionLoader.initialize(CollectionLoader.java:59)
	org.hibernate.persister.collection.AbstractCollectionPersister.initialize(AbstractCollectionPersister.java:587)
	org.hibernate.event.def.DefaultInitializeCollectionEventListener.onInitializeCollection(DefaultInitializeCollectionEventListener.java:83)
	org.hibernate.impl.SessionImpl.initializeCollection(SessionImpl.java:1743)
	org.hibernate.collection.AbstractPersistentCollection.initialize(AbstractPersistentCollection.java:366)
	org.hibernate.collection.AbstractPersistentCollection.read(AbstractPersistentCollection.java:108)
	org.hibernate.collection.AbstractPersistentCollection.readSize(AbstractPersistentCollection.java:131)
	org.hibernate.collection.PersistentBag.size(PersistentBag.java:248)
	org.oscarehr.common.model.ProviderPreference.hibernatePreFetchCollectionsFix(ProviderPreference.java:196)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:498)
	org.hibernate.ejb.event.BeanCallback.invoke(BeanCallback.java:23)
	org.hibernate.ejb.event.EntityCallbackHandler.callback(EntityCallbackHandler.java:80)
	org.hibernate.ejb.event.EntityCallbackHandler.postLoad(EntityCallbackHandler.java:73)
	org.hibernate.ejb.event.EJB3PostLoadEventListener.onPostLoad(EJB3PostLoadEventListener.java:33)
	org.hibernate.engine.TwoPhaseLoad.initializeEntity(TwoPhaseLoad.java:221)
	org.hibernate.loader.Loader.initializeEntitiesAndCollections(Loader.java:877)
	org.hibernate.loader.Loader.doQuery(Loader.java:752)
	org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	org.hibernate.loader.Loader.loadEntity(Loader.java:1881)
	org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:71)
	org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:65)
	org.hibernate.persister.entity.AbstractEntityPersister.load(AbstractEntityPersister.java:3072)
	org.hibernate.event.def.DefaultLoadEventListener.loadFromDatasource(DefaultLoadEventListener.java:434)
	org.hibernate.event.def.DefaultLoadEventListener.doLoad(DefaultLoadEventListener.java:415)
	org.hibernate.event.def.DefaultLoadEventListener.load(DefaultLoadEventListener.java:165)
	org.hibernate.event.def.DefaultLoadEventListener.proxyOrLoad(DefaultLoadEventListener.java:223)
	org.hibernate.event.def.DefaultLoadEventListener.onLoad(DefaultLoadEventListener.java:126)
	org.hibernate.impl.SessionImpl.fireLoad(SessionImpl.java:905)
	org.hibernate.impl.SessionImpl.get(SessionImpl.java:842)
	org.hibernate.impl.SessionImpl.get(SessionImpl.java:835)
	org.hibernate.ejb.AbstractEntityManagerImpl.find(AbstractEntityManagerImpl.java:182)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:498)
	org.springframework.orm.jpa.SharedEntityManagerCreator$SharedEntityManagerInvocationHandler.invoke(SharedEntityManagerCreator.java:240)
	com.sun.proxy.$Proxy101.find(Unknown Source)
	org.oscarehr.common.dao.AbstractDao.find(AbstractDao.java:145)
	org.oscarehr.common.dao.AbstractDao$$FastClassByCGLIB$$d259da96.invoke(<generated>)
	net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191)
	org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)
	org.oscarehr.common.dao.ProviderPreferenceDao$$EnhancerByCGLIB$$c53e174c.find(<generated>)
	oscar.login.LoginAction.execute(LoginAction.java:334)
	org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:419)
	org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:224)
	org.apache.struts.action.ActionServlet.process(ActionServlet.java:1194)
	org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:432)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:239)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:215)
	oscar.oscarSecurity.LoginFilter.doFilter(LoginFilter.java:205)
	org.displaytag.filter.ResponseOverrideFilter.doFilter(ResponseOverrideFilter.java:125)
	org.oscarehr.util.LoggedInUserFilter.doFilter(LoggedInUserFilter.java:56)
	org.oscarehr.util.DbConnectionFilter.doFilter(DbConnectionFilter.java:66)
	org.oscarehr.util.UserActivityFilter.doFilter(UserActivityFilter.java:88)
	org.oscarehr.common.printing.PrivacyStatementAppendingFilter.doFilter(PrivacyStatementAppendingFilter.java:121)
	org.oscarehr.util.ResponseDefaultsFilter.doFilter(ResponseDefaultsFilter.java:102)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:199)

Root Cause

java.sql.SQLSyntaxErrorException: Table 'oscar.ProviderPreferenceAppointmentScreenForm' doesn't exist
	com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1003)
	org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	org.apache.commons.dbcp.DelegatingPreparedStatement.executeQuery(DelegatingPreparedStatement.java:96)
	org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:208)
	org.hibernate.loader.Loader.getResultSet(Loader.java:1808)
	org.hibernate.loader.Loader.doQuery(Loader.java:697)
	org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	org.hibernate.loader.Loader.loadCollection(Loader.java:2015)
	org.hibernate.loader.collection.CollectionLoader.initialize(CollectionLoader.java:59)
	org.hibernate.persister.collection.AbstractCollectionPersister.initialize(AbstractCollectionPersister.java:587)
	org.hibernate.event.def.DefaultInitializeCollectionEventListener.onInitializeCollection(DefaultInitializeCollectionEventListener.java:83)
	org.hibernate.impl.SessionImpl.initializeCollection(SessionImpl.java:1743)
	org.hibernate.collection.AbstractPersistentCollection.initialize(AbstractPersistentCollection.java:366)
	org.hibernate.collection.AbstractPersistentCollection.read(AbstractPersistentCollection.java:108)
	org.hibernate.collection.AbstractPersistentCollection.readSize(AbstractPersistentCollection.java:131)
	org.hibernate.collection.PersistentBag.size(PersistentBag.java:248)
	org.oscarehr.common.model.ProviderPreference.hibernatePreFetchCollectionsFix(ProviderPreference.java:196)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:498)
	org.hibernate.ejb.event.BeanCallback.invoke(BeanCallback.java:23)
	org.hibernate.ejb.event.EntityCallbackHandler.callback(EntityCallbackHandler.java:80)
	org.hibernate.ejb.event.EntityCallbackHandler.postLoad(EntityCallbackHandler.java:73)
	org.hibernate.ejb.event.EJB3PostLoadEventListener.onPostLoad(EJB3PostLoadEventListener.java:33)
	org.hibernate.engine.TwoPhaseLoad.initializeEntity(TwoPhaseLoad.java:221)
	org.hibernate.loader.Loader.initializeEntitiesAndCollections(Loader.java:877)
	org.hibernate.loader.Loader.doQuery(Loader.java:752)
	org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:259)
	org.hibernate.loader.Loader.loadEntity(Loader.java:1881)
	org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:71)
	org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:65)
	org.hibernate.persister.entity.AbstractEntityPersister.load(AbstractEntityPersister.java:3072)
	org.hibernate.event.def.DefaultLoadEventListener.loadFromDatasource(DefaultLoadEventListener.java:434)
	org.hibernate.event.def.DefaultLoadEventListener.doLoad(DefaultLoadEventListener.java:415)
	org.hibernate.event.def.DefaultLoadEventListener.load(DefaultLoadEventListener.java:165)
	org.hibernate.event.def.DefaultLoadEventListener.proxyOrLoad(DefaultLoadEventListener.java:223)
	org.hibernate.event.def.DefaultLoadEventListener.onLoad(DefaultLoadEventListener.java:126)
	org.hibernate.impl.SessionImpl.fireLoad(SessionImpl.java:905)
	org.hibernate.impl.SessionImpl.get(SessionImpl.java:842)
	org.hibernate.impl.SessionImpl.get(SessionImpl.java:835)
	org.hibernate.ejb.AbstractEntityManagerImpl.find(AbstractEntityManagerImpl.java:182)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:498)
	org.springframework.orm.jpa.SharedEntityManagerCreator$SharedEntityManagerInvocationHandler.invoke(SharedEntityManagerCreator.java:240)
	com.sun.proxy.$Proxy101.find(Unknown Source)
	org.oscarehr.common.dao.AbstractDao.find(AbstractDao.java:145)
	org.oscarehr.common.dao.AbstractDao$$FastClassByCGLIB$$d259da96.invoke(<generated>)
	net.sf.cglib.proxy.MethodProxy.invoke(MethodProxy.java:191)
	org.springframework.aop.framework.Cglib2AopProxy$CglibMethodInvocation.invokeJoinpoint(Cglib2AopProxy.java:689)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:150)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:110)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	org.springframework.aop.framework.Cglib2AopProxy$DynamicAdvisedInterceptor.intercept(Cglib2AopProxy.java:622)
	org.oscarehr.common.dao.ProviderPreferenceDao$$EnhancerByCGLIB$$c53e174c.find(<generated>)
	oscar.login.LoginAction.execute(LoginAction.java:334)
	org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:419)
	org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:224)
	org.apache.struts.action.ActionServlet.process(ActionServlet.java:1194)
	org.apache.struts.action.ActionServlet.doPost(ActionServlet.java:432)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:239)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:215)
	oscar.oscarSecurity.LoginFilter.doFilter(LoginFilter.java:205)
	org.displaytag.filter.ResponseOverrideFilter.doFilter(ResponseOverrideFilter.java:125)
	org.oscarehr.util.LoggedInUserFilter.doFilter(LoggedInUserFilter.java:56)
	org.oscarehr.util.DbConnectionFilter.doFilter(DbConnectionFilter.java:66)
	org.oscarehr.util.UserActivityFilter.doFilter(UserActivityFilter.java:88)
	org.oscarehr.common.printing.PrivacyStatementAppendingFilter.doFilter(PrivacyStatementAppendingFilter.java:121)
	org.oscarehr.util.ResponseDefaultsFilter.doFilter(ResponseDefaultsFilter.java:102)
	net.bull.javamelody.MonitoringFilter.doFilter(MonitoringFilter.java:199)

**Comment by jw098** [comment]
Wait a minute. I made a dumb mistake. I didn't re-download from github. Let me try again.

**Comment by jw098** [comment]
I tried purging and reinstalling, re-cloning the github repository, but I still get the same error.

But I see there have been no new commits. Did you make changes to the github repository?

**Comment by warrendennis** [comment]
The actual error is this: 

```java.sql.SQLSyntaxErrorException: Table 'oscar.ProviderPreferenceAppointmentScreenForm' doesn't exist```

That database table did not get installed.  Did you see any errors during bootstrap? 

There is no update or patch needed. Everything seems to be in order.  Strange.

Oh wait.  Are you using Docker Compose version 2?  You should be.  The documentation hasnt been fully updated yet.

**Comment by jw098** [comment]
I'm using Docker compose v2.16.0.

I tried reinstalling again. I still get the same error

I don't know how to edit the database. But would that be helpful in this case?

**Comment by warrendennis** [comment]
This error is confirmed.  I'm not sure why, but the bootstrap script is not creating that specific table.  We are working on a fix.

**Comment by warrendennis** [comment]
We just published an update.  Please try to install again from the beginning

**Comment by jw098** [comment]
I tried reinstalling. I no longer get the HTTP Status 500 error. 

This is the new error:

Error Page
Received the exception:
org.apache.jasper.JasperException: An exception occurred processing [/provider/appointmentprovideradminday.jsp] at line [162] 159: } 160: 161: // are prevention stop sign icons being loaded? This needs to be known when loading the schedule 162: pageContext.setAttribute("isPreventionWarningDisabled", providerPreventionManager.isDisabled()); 163: %> 164: <%! 165: //multisite starts ===================== Stacktrace:
An exception occurred processing [/provider/appointmentprovideradminday.jsp] at line [162] 159: } 160: 161: // are prevention stop sign icons being loaded? This needs to be known when loading the schedule 162: pageContext.setAttribute("isPreventionWarningDisabled", providerPreventionManager.isDisabled()); 163: %> 164: <%! 165: //multisite starts ===================== Stacktrace:

**Comment by jw098** [comment]
I tried updating the EMR image. I still get the same error. The terminal tells me that it is up to date.

`latest: Pulling from openosp/open-osp`
`Digest: sha256:1a01365129d8eac8eac4ce0b09c7516e8e6f5444d9a47989aa70e8c4a5aa250f`
`Status: Image is up to date for openosp/open-osp:latest`
`docker.io/openosp/open-osp:latest`

**Comment by warrendennis** [comment]
Strange.

Fortunately there is an easy workaround.  

1). after you're logged in - and the error appears - alter the URL address like this:  

```https://<your ip>/oscar/**administration**```

2). You should get the Administration page.
3). go to: Schedule Management / Prevention Notification Settings
4). select the the Enable or Disabled radio button (which ever is available)
5). click Save
6). Close the Administration window and then log back in.

**Comment by jw098** [comment]
Thanks. Your workaround worked.

I also tried reinstalling from scratch on a fresh Ubuntu install, and it worked as well.
